app/views/shared/_cancel.html.slim

```diff
@@ -1,4 +1,6 @@
 - cancel = sign_up_or_idv_no_js_link || link
+- decorated_session = decorated_session
```

Comment:
This is not needed because the helper method `decorated_session` is already defined in `application_controller`. A `helper_method` defined in application controller is available in all views.